code/game/objects/items/ornaments.dm

```diff
@@ -0,0 +1,187 @@
+/obj/item/ornament
+	name = "ornament"
+	desc = "A glass ornament. What some would call plain, others call elegant."
+	icon = 'icons/obj/ball_ornaments.dmi'
+	icon_state = "white_ball_ornament"
+	w_class = 2
+
+/obj/item/ornament/attackby(obj/item/weapon/W, mob/user)
+	..()
+	if(istype(src, /obj/item/ornament/teardrop))
+		return
+	if(istype(W, /obj/item/toy/crayon) || istype(W, /obj/item/weapon/pen))
+		var/obj/item/ornament/O
+		var/color_string = null
+		if(istype(W, /obj/item/toy/crayon))
+			var/obj/item/toy/crayon/C = W
+			switch(C.colourName)
+				if("red")
+					O = new /obj/item/ornament/red(get_turf(src))
```

Comment:
Unless you mean keep the children but just don't use them for this, which seems a little strange to me, but I wouldn't mind.